.circleci/config.yml

```diff
@@ -277,6 +277,32 @@ jobs:
             ctest -V -Ddashboard_source_config_dir:PATH="Wrapping/Java" -S "${CTEST_SOURCE_DIRECTORY}/.circleci/circleci.cmake"
       - *junit-fotmatting
       - *junit-store-test-results
+  sphinx-docs:
+    <<: *defaults
+    environment:
+      <<: *default_environment_keys
+      SPHINX_SOURCE_DIR: /home/circleci/SimpleITK/docs/source
+      SPHINX_BUILD_DIR: /tmp/docs
+      SPHINX_OPTS: -aE
+    steps:
+      - checkout:
+          path : ~/SimpleITK
+      - run:
+          name: Create Python Environment
+          command: |
+              python -m pip install --user --upgrade pip
+              python -m pip install --user "sphinx<2" "sphinx-rtd-theme<0.5"
```

Comment:
Should the above packaged be moved to the requirements.txt file?

Reply:
yeah, it should probably be in `docs/requirements.txt`